games-fps/chocolate-doom/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<longdescription lang="en">
+		Chocolate Doom aims to accurately reproduce the original DOS version of Doom and other games based on the Doom engine in a form that can be run on modern computers.
+	</longdescription>
+	<maintainer type="person">
+		<email>vilhelm.gray@gmail.com</email>
+		<name>William Breathitt Gray</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>games@gentoo.org</email>
+		<name>Gentoo Games Project</name>
+	</maintainer>
+	<upstream>
+		<bugs-to>https://github.com/chocolate-doom/chocolate-doom/issues</bugs-to>
+		<doc lang="en">https://www.chocolate-doom.org</doc>
+		<remote-id type="github">chocolate-doom/chocolate-doom</remote-id>
+	</upstream>
+</pkgmetadata>
```

Comment:
It is totally not obvious what the `python` USE flag actually does. Ideally we would have split this out into `bash-completion`, `doc`, `fonts`, and `icons` flags or something but the build system makes it all or nothing so please explain that the `python` flag enables those things.

Reply:
Upstream PR opened: https://github.com/chocolate-doom/chocolate-doom/pull/1294

I'll add that patch to the Gentoo package if the Chocolate Doom team accepts it.